__tests__/articles.test.js

```diff
@@ -61,6 +61,7 @@ describe("get all articles", () => {
           created_at: expect.any(String),
           votes: expect.any(Number),
           article_img_url: expect.any(String),
+          comment_count: expect.any(Number)

```

Comment:
Your SQL looks right to me but again - if we aren't checking the exact value of comment_count for article 1 is exactly correct as in the data (10) then this is not a reliable enough test